src/MustardBlack.Assets.Build/Program.cs

```diff
@@ -85,8 +86,7 @@ static string ProcessJs(string options, IAssetLoader assetLoader, string path)
 
 				if (opts.Contains("minify"))
 				{
-					javascriptPreprocessor = new YuiJavascriptPreprocessor();
-					result = javascriptPreprocessor.Process(new [] { new AssetContent("<babel-output>", result) });
+					result = Uglify.Js(result).Code;

```

Comment:
For consistency, does this want to be included as a `Preprocessor` in a similar light to the current YuiCompressor. That way we maintain some level of project hierarchy and removes the need to include the package references in the build project. Just Project references.

We might want to look into making this Uglify a forward compatible option. eg. `opts.Contains("minify-nuglify")` that way we can consciously make the change over without potentially affecting all of V5/Admin

Reply:
I'll redo this as a preprocessor.

This won't touch V5 or Admin as they don't use Babel. The only risk here is to SelfService. I'll see about adding this as an option though.